packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.types.ts

```diff
@@ -6,7 +6,7 @@ import { DirectionalHint } from '../../common/DirectionalHint';
 import { IRefObject, IStyleFunctionOrObject } from '../../Utilities';
 import { IStyle, ITheme } from '../../Styling';
 
-export interface ITooltipHost { }
+export interface ITooltipHost {}
```

Comment:
I'm not familiar familiar with this code so this might be obvious but what does this do for adding a label?